core/library.go

```diff
@@ -851,10 +851,7 @@ func (handler *graphMutatorHandler) ResolveDependencySortMutator(mctx abstr.Bott
 	if _, ok := mainModule.(*defaults); ok {
 		return // ignore bob_defaults
 	}
-
-	mainModuleName := buildbpName(mainModule.Name())
```

Comment:
Could we keep the hoist of mainModuleName?

Reply:
Sure, I don't see why not :)